src/content/reference/web-interface/app-catalog.md

```diff
@@ -0,0 +1,46 @@
++++
+title = "The Giant Swarm App Catalog in The Web Interface"
+description = "What the Giant Swarm App Catalog looks like on our Web Interface and how to use it."
+date = "2019-07-05"
+layout = "subsection"
+weight = 10
++++
+
+# The Giant Swarm App Catalog in The Web Interface
+
+This page will give you an overview of how to do some common tasks related to the
+Giant Swarm App Catalog using our Web Interface.
+
+Go here for an [overview of the Giant Swarm App Catalog](/basics/app-catalog/) instead.
+
+### Viewing all App Catalogs
+
+Our web interface lets you browse the App Catalogs installed on your Control Plane.
+Click on "Apps" in the navigation menu. This link will only be visible if your
+Control Plane has at least one App Catalog installed on it.
+
+The screenshot below shows this page when only the "Helm Stable" and
```

Comment:
I don't understand what you are trying to convey with this sentence.
Will the page only show up in that one case, or will it only look that way in said case.

Reply:
The link will show up whenever there is one or more app catalogs installed. 

and THEN when you click on that link this is what that page would look like with two app catalogs installed.

I'll try to make that clearer.